osbuild/api.py

```diff
@@ -215,3 +215,19 @@ def data_to_file():
             "metadata": 0
         }
         client.send(msg, fds=[f.fileno()])
+
+
+def default_main(schema_1: Optional[str] = None, schema_2: Optional[str] = None) -> None:
+    """Provides a default 'main' implementation for osbuild modules. This
+    registers `--schema-1` and `--schema-2` arguments to print available
+    schemas."""
+
+    # This is implemented *very* naively initially.
```

Comment:
What is the reason for implementing this naively?

Reply:
The naive refers to checking `sys.argv` quite directly, if the plan is good then this should be followed up (in this PR) with proper argument parsing.

Reason was to get the PoC done.